README.md

```diff
@@ -76,6 +81,19 @@ using an array of gvcfs as input.*
   The dynamic scatter interval creating was optimized for genomes.  The scattered SNP 
   VariantRecalibration may fail because of two few "bad" variants to build the negative model. 
   Also, apologies that the logging for SNP recalibration is overly verbose.
+- No allele subsetting for the JointGenotyping workflow
+  - for large cohorts, even exome callsets can have more than 1000 alleles at low 
+    complexity/STR sites
+  - for sites with more than 6 alternate alleles (by default) called genotypes will be returned, 
+    but without the PLs since the PL arrays get enormous
+  - allele-specific filtering could be performed if AS annotations are present, 
+    but the data will still be in the VCF in one giant INFO field
+- JointGenotyping output is divided into lots of shards
+  - desirable for use in [Hail](https://hail.is/), which supports parallel import
+  - Its possible to use [GatherVcfs](https://gatk.broadinstitute.org/hc/en-us/search?utf8=%E2%9C%93&query=GatherVcfs) to combine shards per chromosome.
```

Comment:
I would take out "per chromosome" because the user has to decide that, it isn't in the workflow.